package.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "network-dashboard",
+  "description": "Aragon Network Dashboard",
+  "version": "0.1.0",
+  "private": true,
+  "license": "AGPL-3.0-or-later",
+  "author": "Aragon Association <legal@aragon.org>",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/aragon/network-dashboard.git"
+  },
+  "bugs": {
+    "url": "https://github.com/aragon/network-dashboard/issues"
+  },
+  "homepage": "https://github.com/aragon/network-dashboard#readme",
+  "keywords": [
+    "aragon",
+    "network",
+    "ethereum",
+    "dapp",
+    "blockchain",
+    "startups"
+  ],
+  "engines": {
+    "node": ">= 12.0.0"
+  },
+  "bundlewatch": {
+    "files": [
+      {
+        "path": "public/*.js",
+        "maxSize": "1600kb",
+        "compression": "gzip"
+      }
+    ]
+  },
+  "dependencies": {
+    "@aragon/ui": "^1.5.0",
+    "core-js": "^3.3.2",
+    "dotenv": "^8.2.0",
```

Comment:
Minor nitpick, but I think we would count `dotenv` and `rimraf` to be devdependencys?

Reply:
Good spot!